docs/cli-mode.md

```diff
@@ -283,4 +313,27 @@ Flags:
 ```
 The ```--config``` flag can be used to provide the path to the env specific config folder that contains the ```serverConfig.json``` and ```toolConfig.json``` files with the details of the environment to which the resources should be imported. If the flag is not provided, the tool looks for the server configurations in the environment variables.
 
-The ```--inputDir``` flag can be used to provide the path to the local directory where the resource configuration files are stored. If the flag is not provided, the tool looks for the resource configuration files at the current working directory.
\ No newline at end of file
+The ```--inputDir``` flag can be used to provide the path to the local directory where the resource configuration files are stored. If the flag is not provided, the tool looks for the resource configuration files at the current working directory.
+
+## Supported Resource Types
+The tool supports the following resource types:
+
+### Applications
+The tool supports exporting and importing applications. The exported application configuration files can be found under the ```Applications``` folder in the local directory. If it is required to deploy a new application through the import command of the tool, the new file should be placed under the ```Applications``` folder in the local directory.
+
+Since the system applications: ```Console``` and ```My Account``` are made read-only in WSO2-IS, if it is required to update the console and myaccount applications through the import command, add the following to the ```deployment.toml``` of the WSO2 Identity Server and restart the server.
```

Comment:
```suggestion
Since the system applications: ```Console``` and ```My Account``` are read-only applications in WSO2 Identity Server, if it is required to update these applications through the `import command`, add the following configurations to the ```deployment.toml``` file and restart the server.
```